openedx/core/djangoapps/credit/urls.py

```diff
@@ -1,43 +1,53 @@
 """
 URLs for the credit app.
 """
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
+from rest_framework.routers import DefaultRouter
 
-from .api.provider import get_credit_provider_info
-from .views import create_credit_request, credit_provider_callback, get_providers_detail, get_eligibility_for_user
+from openedx.core.djangoapps.credit import views
+from openedx.core.djangoapps.credit.api.provider import get_credit_provider_info
 
 PROVIDER_ID_PATTERN = r'(?P<provider_id>[^/]+)'
 
-urlpatterns = patterns(
+V1_URLS = patterns(
     '',
 
     url(
-        r"^v1/providers/(?P<provider_id>[^/]+)/$",
-        get_credit_provider_info,
-        name="get_provider_info"
+        r'^providers/$',
+        views.get_providers_detail,
+        name='providers_detail'
     ),
+
     url(
-        r"^v1/providers/$",
-        get_providers_detail,
-        name="providers_detail"
+        r'^providers/{provider_id}/$'.format(provider_id=PROVIDER_ID_PATTERN),
+        get_credit_provider_info,
+        name='get_provider_info'
     ),
 
     url(
-        r"^v1/providers/{provider_id}/request/$".format(provider_id=PROVIDER_ID_PATTERN),
-        create_credit_request,
-        name="create_request"
+        r'^providers/{provider_id}/request/$'.format(provider_id=PROVIDER_ID_PATTERN),
+        views.create_credit_request,
+        name='create_request'
     ),
 
     url(
-        r"^v1/providers/{provider_id}/callback/?$".format(provider_id=PROVIDER_ID_PATTERN),
-        credit_provider_callback,
-        name="provider_callback"
+        r'^providers/{provider_id}/callback/?$'.format(provider_id=PROVIDER_ID_PATTERN),
+        views.credit_provider_callback,
+        name='provider_callback'
     ),
 
     url(
-        r"^v1/eligibility/$",
-        get_eligibility_for_user,
-        name="eligibility_details"
+        r'^eligibility/$',
+        views.get_eligibility_for_user,
+        name='eligibility_details'
     ),
+)
+
+router = DefaultRouter()
+router.register(r'courses', views.CreditCourseViewSet)
+V1_URLS += router.urls
 
+urlpatterns = patterns(
+    '',
+    url(r'^v1/', include(V1_URLS)),
```

Comment:
Speaking of which, is there a reason not to organize this with a package structure similar to that used for the commerce API (i.e., `/credit/api/urls.py` which includes `/credit/api/v1/urls.py`)